.env.development

```diff
@@ -10,7 +10,7 @@ DB_NAME='bhima_test'
 # session variables
 SESS_SECRET='XopEn BlowFISH'
 
-DEBUG=app,errors
+DEBUG=app
```

Comment:
It's helpful to have the errors here so we can see what went wrong on Travis.  Could you restore this?